armi/physics/fuelCycle/fuelHandlerInterface.py

```diff
@@ -90,9 +90,7 @@ def interactEOC(self, cycle=None):
         # keep track of the EOC time in years.
         self.cycleTime[cycle] = timeYears
         runLog.extra(
-            "There are {} assemblies in the Spent Fuel Pool".format(
-                len(self.r.core.sfp)
-            )
+            "There are {} assemblies in the Spent Fuel Pool".format(len(self.r.sfp))
```

Comment:
A bit off topic, but I'll ask anyway. Feel free to punt. 

Before the transition to ruff, pylint used to complain about using `.format()` instead of f-strings. Is this a rule we want to enforce moving forward, or no? 

https://pylint.pycqa.org/en/latest/user_guide/messages/convention/consider-using-f-string.html.

Reply:
Yeah, as a rule we like f-strings better than the various other options.

Here, I didn't create the original `.format()`, of course.  So I was just making the necessary change.  But I can clean that up if you like.

Reply:
For sure. I suppose I was mostly curious if we should add the preference of f-strings over `format.()` into our linting? Recognizing that this will result in a gazillion changes since there are LOTS of `format.()` calls in ARMI.... What do you think?

Reply:
fixed

Reply:
I'm not going to open a 200-file PR for the `.format()` thing, I think.

I mean, maybe if I could script it.

But even then, sometimes people have a LOT of code in their `.format()` statements, and the f-strings can only do extremely simple expressions, so you'd have to create temporary variables.  So there _might_ be a couple format statements that end up looking cleaner.  Maybe? I'm just spit-balling.

Reply:
Ok, I'm fine with it. I don't have a strong preference. Was just an inconsistency I realized. Figured I'd call it out.